frontend/src/flashcards/FlashcardBackPreview.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled"
+
+class FlashcardBack extends React.Component {
+  render() {
+    const Contain = styled.div`
```

Comment:
`FlashcardContainer`? The name will help if you ever need to debug with react-dev-tools.